PlayBook/src/getting_started/troubleshoot_login.md

```diff
@@ -1,4 +1,8 @@
-# Extra Install Instructions for Genshin Impact
+# Troubleshoot App Login
+
+You may encounter some issue when logging in to some apps and games like Genshin Impact. Follow these instructions to fix that. As always, if you run into issues, ask for help on [Discord](https://discord.gg/rMv5qxGTGC).
```

Comment:
"issues"

Reply:
"You may encounter some issue". It should either say "issues" or "You may encounter an issue".